hips/tiles/tests/test_tile.py

```diff
@@ -0,0 +1,48 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.utils.data import get_pkg_data_filename
+from ..description import HipsDescription
+from ..tile import HipsTile
+from pathlib import Path
+import numpy as np
+
+class TestHipsTile:
+    @classmethod
+    def setup_class(cls):
+        filename = get_pkg_data_filename('data/properties.txt')
+        hips_description = HipsDescription.read(filename)
+
+        cls.tile = HipsTile(hips_description, 6, 24185, 'jpg', 512)
```

Comment:
I suggest to call this function with keyword arguments, i.e. `order=6`, ... instead of just `6`, to make it easier to read, i.e. know what the parameters are without having to look up the class docs.
If the lines get to long, wrap them (Python is OK with this within parentheses).